main.cpp

```diff
@@ -52,85 +52,93 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: Fork
+//  action 1: the fork holds food
+fork.holds();
+//  action 2: the fork pushes food forwards
+fork.pushes();
+//  action 3: the fork pulls food backwards
+fork.pulls();
 
-//  action 2:
-
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: Monkey
+//  action 1: the monkey climbs 
 
-//  action 2:
+//  action 2: the monkey sits
 
-//  action 3:
+//  action 3:the monkey falls asleep
  
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: Cat 
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat chases mice
+cat.chases();
+//  action 3: the cat annoys owner
+cat.annoys();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.